### PR TITLE
#793 - Explorer: "View Owners Strategies"

### PR DESCRIPTION
**Pull request by pingustar**
#793

### CODE REVIEW

**Comment by ashachaf** [comment]
- [ ] search input field keeps the wallet address on browser "back"
flow:
1. visit explorer
2. select pair
3. search for dai/eth
4. click the menu of any strategy and visit owner
5. click the back on the browser to go back to the last page
expected: see the pair in the search input field
actual: see the wallet address
image

File: src/components/strategies/overview/strategyBlock/StrategyBlockManage.tsx [hunk 1]
```diff
@@ -230,39 +255,43 @@ const ManageItem: FC<{
...
+  disabled?: boolean;
+}> = ({ title, id, setManage, action, isExplorer, disabled }) => {
   const tooltipText = getTooltipTextByStrategyEditOptionsId(isExplorer)?.[id];
   const { belowBreakpoint } = useBreakpoints();
 
+  const Content = () => {
+    return (
+      <div
+        onClick={() => {
+          if (disabled) return;
```
Comment (GrandSchtroumpf) [inline src/components/strategies/overview/strategyBlock/StrategyBlockManage.tsx:267]: You don't need this if you set style `pointer-events: none`

File: src/components/strategies/overview/strategyBlock/StrategyBlockManage.tsx [hunk 2]
```diff
@@ -230,39 +255,43 @@ const ManageItem: FC<{
...
+
+          action && action();
+          setManage(false);
+        }}
+        className={cn('rounded-6 p-12', {
+          'cursor-pointer': !disabled,
+          'cursor-not-allowed': disabled,
```
Comment (GrandSchtroumpf) [inline src/components/strategies/overview/strategyBlock/StrategyBlockManage.tsx:274]: Common behavior for disabled button is usually "pointer-events: none", like that you don't need to verify `if (disabled)` above

**Review by GrandSchtroumpf — APPROVED** [review]
Code looks good. Some small html/css suggestion

**Review by zavelevsky — APPROVED** [review]
LGTM

**Review by tiagofilipenunes — APPROVED** [review]
LGTM

File: src/components/strategies/overview/strategyBlock/StrategyBlockManage.tsx [hunk 3]
```diff
@@ -230,39 +255,43 @@ const ManageItem: FC<{
...
   isExplorer?: boolean;
-}> = ({ title, id, setManage, action, isExplorer }) => {
+  disabled?: boolean;
+}> = ({ title, id, setManage, action, isExplorer, disabled }) => {
   const tooltipText = getTooltipTextByStrategyEditOptionsId(isExplorer)?.[id];
   const { belowBreakpoint } = useBreakpoints();
 
+  const Content = () => {
+    return (
+      <div
```
[the remark this answers is not in the document]
Comment (pingustar) [reply src/components/strategies/overview/strategyBlock/StrategyBlockManage.tsx:265]: thanks, changed to button

**Review by ashachaf — APPROVED** [review]

**Pull request merged**